service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/container/CopyAzureStorageContainerDefinitionStepTest.java

```diff
@@ -134,9 +133,6 @@ void doStep_clonesDefinition() throws InterruptedException {
     assertEquals(cloned.getName(), destResourceName);
     assertEquals(cloned.getCloningInstructions(), CloningInstructions.COPY_DEFINITION);
     assertEquals(sharedAccountRegion, cloned.getRegion());
-    assertNull(
-        cloned.getStorageAccountId(),
-        "Storage account should be null since this is a landing zone backed workspace");
     assertEquals(sharedAccountRegion, cloned.getRegion());
```

Comment:
This line duplicates 135. I'm also wondering what `sharedAccountRegion` refers to here.

Reply:
`sharedAccountRegion` is the region of the storage account